data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```diff
@@ -62,6 +62,11 @@
       "examples": [
         "net.corda.notary.MyNotaryService"
       ]
+    },
+    "corda.auth.token": {
+      "description": "Optional. A UUID string that uniquely identities the pre-auth token.",
+      "type": "string",
+      "pattern": "[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$"
```

Comment:
This could be more specific. Here's the regex we use for the UUID in the group policy file and the `UUID_REGEX` property in `RbacKeys.kt`:

`^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89aAbB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$`

Source:
https://stackoverflow.com/a/14166194